CMakeLists.txt

```diff
@@ -0,0 +1,48 @@
+cmake_minimum_required (VERSION 2.6)
+project (LIBUAST C CXX)
+
+# C++ 11 required
+set (CMAKE_CXX_STANDARD 11)
+
+# Required dependencies
+find_package(ZLIB REQUIRED)
```

Comment:
It works for me without direct `ZLIB` and `LibLZMA` dependency tracking (since it's handled by `LibXml2`